### ISSUE TITLE
Bug: Testing framework seg faults

### ISSUE THREAD

**Issue by acquamarin** [kuzudb/kuzu#4432]
### Kùzu version

master

### What operating system are you using?

MAC M1

### What happened?

I have this testing file:
```
-DATASET CSV tinysnb

--

-CASE Comment


-STATEMENT CALL show_tables() RETURN *
---- ok
```
If i run this testing file, it seg faults.

**Comment by WWW0030** [comment]
Looks like a problem with the `CASE` name:
In https://github.com/kuzudb/kuzu/blob/29967cae831c05b6898cb64075d242a8ad3f6cfe/test/test_files/comment/comment.test
comment is used as a case name already. It looks like that is causing the problem, as changing the file above with something like -CASE CommentWithSuffix passes.

**Comment by acquamarin** [comment]
```
-DATASET CSV tinysnb

--

-CASE dkkdsad


-STATEMENT CALL show_tables() RETURN *
---- ok
```
Still seg faults with a random unique string

**Comment by acquamarin** [comment]
Solved in #4445
